roles/services/tasks/main.yml

```diff
@@ -38,14 +39,23 @@
   with_items:
     - layer-categories-hbg.timer
     - layer-categories-hbg.service
+    - layer-categories-lbg.timer
+    - layer-categories-lbg.service
 
-- name: Enable and start layer-categories.timer
+- name: Enable and start Herrenberg layer-categories.timer
   ansible.builtin.systemd:
     name: layer-categories-hbg.timer
     enabled: true
     state: started
     daemon_reload: yes
 
+- name: Enable and start Ludwigsburg layer-categories.timer
```

Comment:
This can be parameterize with `with_items`. There are quite a few example of this in this repo.